### ISSUE TITLE
The tooltips are not getting appeared while keyboard user is traversing the toolbar buttons of the Xamarin Inspector window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#289]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Traverse to the below Toolbar menu buttons:
Refresh button, 
Select a UI element to inspect in the running application"  button, 
Show REPL view button, 
Show Inspector view button
3. Observe the tooltip while traversing using Keyboard.

### Expected Behavior
Tool tips should be appeared while keyboard user is traversing these buttons.
### Actual Behavior
The tooltips are not getting appeared while keyboard user is traversing the toolbar buttons of the Xamarin Inspector window.
Note1: The tooltip is getting appeared while user hovers the mouse focus on these buttons.
Note2: VoiceOver is also reading the name/purpose of these buttons, so no issue is being observed for VoiceOver AT. 
### User Impact
The keyboard user may find it difficult to understand the purpose of the buttons until tooltip is not appeared.

![kb5_toolbar_notool tips](https://user-images.githubusercontent.com/22956130/39986530-922a3692-577f-11e8-843c-cb4662e7ba89.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
